exercises/grade-school/.meta/DONT-TEST-STUB

```diff
@@ -0,0 +1,2 @@
+No type signatures on functions (TODO: Should we have them?),
```

Comment:
if you believe we should have them, I will gladly add them and remove this `DONT-TEST-STUB`